tests/integration/devfile/cmd_devfile_env_test.go

```diff
@@ -43,14 +53,24 @@ var _ = Describe("odo devfile env command tests", func() {
 			}
 			helper.MatchAllInOutput(output, wantOutput)
 		})
+
+		It("should show all default parameters with odo view info and JSON output ", func() {
+			output := helper.Cmd("odo", "env", "view", "-o", "json").ShouldPass().Out()
+			values := gjson.GetMany(output, "kind", "spec.name", "spec.project", "spec.appName")
+			expected := []string{"EnvInfo", "acomponentname", commonVar.Project, "app"}
+			Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+		})
 	})
 
-	Context("When executing env set and unset", func() {
-		It("Should successfully set and unset the parameters", func() {
+	When("executing env set", func() {
+		BeforeEach(func() {
 			helper.Cmd("odo", "create", "nodejs").ShouldPass()
```

Comment:
Why is this `When` not set inside `When("creating a component")` considering the first step here is creating a component?

Reply:
My bad, good catch